Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:14.04
+MAINTAINER Seth Fowler <seth.fowler@barefootnetworks.com>
+
+# Install dependencies.
+RUN apt-get update && \
```

Comment:
We'll be adding bmv2 as a dependence to run the tests. But we need to checkout the source and build it since we don't have ubuntu packages.